packages/paste-core/components/display-pill-group/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@twilio-paste/display-pill-group",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "production",
+  "description": "Used to display a collection of labels.",
```

Comment:
Is "labels" the least confusing language we can use here? Would something like "objects" or "entities" be too esoteric? The term "label" leans too close to Badge to me.

Reply:
Yeah I kind of winged this, open to suggestions!

Reply:
I liked the advise @katieporter gave in the Badge docs for the description. The format for all description are aligning around:

`A(n) [component name] is a(n) [type of element] that does [function].`

> A Display Pill Group is a collection of Pills that are used to visually represent a collection of entities outside of a data entry form

or soemthing